### DISCUSSION TITLE
Need expand/collapse all feature in designer and history runs!

### DISCUSSION THREAD

**Discussion by ajsparos** [Azure/logicapps#1060] [Feature Ideas/Requests]
When editing logic apps or viewing history runs everything is expanded by default. For me this measn I have to collapse all scopes and conditions one at a time to gain overview.

**Comment by james-seddon-paymentsense** [comment]
Would also like this, such a painful experience when having to constantly expand massive logic apps

**Comment by divyaswarnkar** [comment]
@ajsparos @james-seddon-paymentsense  - thanks for your feedback. We released designer update a while back providing ability to expand/collapse all the nodes in a single click.  Below is a screenshot and the icon in red box is the collapse/expand button

<img width="1965" height="1074" alt="image" src="https://github.com/user-attachments/assets/40ea4cac-5dde-48cc-b650-e2e1f0de73fe" />
